src/main/java/nom/tam/image/StandardImageTiler.java

```diff
@@ -74,15 +71,15 @@ public static long getOffset(int[] dims, int[] pos) {
      * @param start   The starting corner values.
      * @param current The current offsets.
      * @param lengths The desired dimensions of the subset.
+     * @param steps   The desired number of steps to take until the next position.
      * @return <code>true</code> if the current array was changed
      */
-    protected static boolean incrementPosition(int[] start, int[] current, int[] lengths) {
-
+    protected static boolean incrementPosition(int[] start, int[] current, int[] lengths, int[] steps) {
```

Comment:
Ooops, I did miss this one before. It's another one where we should keep the old method signature as well as adding the new one.... The rest looks good to go :-)